quickwit-search/src/root.rs

```diff
@@ -103,6 +104,7 @@ pub async fn root_search(
             split_uris: jobs.iter().map(|job| job.split.clone()).collect(),
```

Comment:
so these are split_id, not split_uris right?

We should probably rename?

Reply:
Yes, I’ll rename to split_ids.